samples/XCT.Sample.WPF/MainWindow.xaml.cs

```diff
@@ -14,6 +14,7 @@ public MainWindow()
 			InitializeComponent();
 			Forms.Forms.Init();
 			PancakeViewRenderer.Init();
+			DrawingViewRenderer.Init();
```

Comment:
Sadly this a `no-go`. We can't have any `Init` call on XCT

Reply:
Is it a bad practice or any violation in XCT?
As you can see it is used in PancakeView. Why can’t we do the same?

Reply:
PacakeView isn't part of XCT.
So this is an agreement that we have on this project, that we want all devs to be able to use it without adding any other line of code. Like plug and play lib.

Reply:
I moved it to the Sample project. Init method is removed